test/client_test.js

```diff
@@ -44,6 +62,7 @@ describe('Client', () => {
         authorizeUrl: 'https://sb-auth.smarthealthit.org/authorize',
         tokenUrl: 'https://sb-auth.smarthealthit.org/token',
         registerUrl: 'https://sb-auth.smarthealthit.org/register',
+        launchRegisterUrl: 'https://sb-auth.smarthealthit.org/launch-registration',
```

Comment:
You didn't add these, but I feel like the urls in our fixtures should be more fake, and preferably match the `baseUrl` we're using in the tests.

Reply:
I could go either way here...